wordle-web/components/UserNameDialog.vue

```diff
@@ -0,0 +1,28 @@
+<!-- gets the username-->
+<template>
+    <v-dialog v-model="modelValue" max-width="500" persistent>
+        <v-card>
+            <v-sheet color="primary"></v-sheet>
+            <v-card-title>Enter your user name current user is {{ userName }}</v-card-title>
```

Comment:
I would take out the "current user is _" part, since it's already displayed in the top right, and maybe add a colon. I.e "<v-card-title>Enter your username: </v-card-title>"

Reply:
I also agree, this would be much cleaner. We'll implement this right away.